packages/circuit-ui/components/ComparisonTable/utils.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2025, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { FeatureSection } from './components/PlanTable/PlanTable.js';
+
+export function getFirstEightRows(arr: FeatureSection[]): FeatureSection[] {
```

Comment:
The number of rows could change in the future, so I wouldn't hard-code it in the name.